src/storage/FirebaseStorageDirective.js

```diff
@@ -0,0 +1,32 @@
+/* istanbul ignore next */
+(function () {
+  "use strict";
+
+  function FirebaseStorageDirective($firebaseStorage, firebase) {
+    return {
+      restrict: 'A',
+      priority: 99, // run after the attributes are interpolated
+      scope: {},
+      link: function (scope, element, attrs) {
+        // $observe is like $watch but it waits for interpolation
+        // Ex: <img firebase-src="{{ myUrl }}"/>
+        attrs.$observe('firebaseSrc', function (newFirebaseSrcVal) {
+          if (newFirebaseSrcVal !== '' &&
```

Comment:
Also, if the passed value of `newFirebaseSrcVal` is invalid, do we want to throw an error? Seems like we are just silently failing in that case which will be hard for developers to track down.